bin/sanitycheck.js

```diff
@@ -184,6 +184,7 @@ var KNOWN_WARNINGS = [
   "App pongclock file pongclock.settings.js should be evaluated as a function but doesn't end in ')'",
   "App popconlaunch file popcon.settings.js should be evaluated as a function but doesn't end in ')'",
   "App poweroff file poweroff.settings.js should be evaluated as a function but doesn't end in ')'",
+  "App promenu file promenu.settings.js should be evaluated as a function but doesn't end in ')'",
```

Comment:
I'm not sure about this one. I think you should just drop the semi-colon at the end of the settings file.

Reply:
Previous discussion with @bobrippling here: https://github.com/espruino/BangleApps/pull/3627#discussion_r1819705496

Reply:
Ah, I only ran `npm run build` and not `build.sh`, that should fix it! I'll update the PR accordingly tomorrow

Reply:
Nice, yeah that's exactly it - and thanks for the improvements!